crates/bevy_reflect/src/reflect.rs

```diff
@@ -6,10 +6,17 @@ use crate::{
 use std::{
     any::{self, Any, TypeId},
     fmt::Debug,
+    hash::BuildHasher,
 };
 
 use crate::utility::NonGenericTypeInfoCell;
-pub use bevy_utils::AHasher as ReflectHasher;
+
+use bevy_utils::FixedState;
+
+#[inline]
+pub fn reflect_hasher() -> bevy_utils::AHasher {
```

Comment:
Nit: This should probably have a doc stating what it does, how/where it's used, and when/why to use it .

Reply:
Also done :)